doc/lua_api.md

```diff
@@ -1751,6 +1751,15 @@ Displays a horizontal bar made up of half-images with an optional background.
 * `item`: Position of item that is selected.
 * `direction`: Direction the list will be displayed in
 * `offset`: offset in pixels from position.
+* `alignment`: The alignment of the inventory.
+  It starts at the top left corner and not at the center like the most other types.
+
+### `hotbar`
+
+* `direction`: Direction the list will be displayed in
+* `offset`: offset in pixels from position.
+* `alignment`: The alignment of the inventory.
+  It starts at the top left corner and not at the center like the most other types.
```

Comment:
(The position when drawing right-to-left is broken anyway, see the first comment "How does the PR work?".)

I choose the top left corner, to comply with the deprecated behavior, when elements of type `inventory` didn't support an alignment. i.e.
- In the past: it could only be aligned at the top left corner, and the aligned parameter didn't change anything.
- After this PR: With no aligned parameter i.e. `(0,0)`, It is still aligned at the top left corner.

I figured it may be better to check this when the element gets read from Lua, so set the alignment to `(-1,-1)` when `alignment = nil`, and make it central aligned at `(0,0)` like all other elements.
I will change like this if there is no objection.